addons/calendar/models/res_users.py

```diff
@@ -10,6 +10,15 @@
 class Users(models.Model):
     _inherit = 'res.users'
 
+    calendar_default_privacy = fields.Selection(
+        related='res_users_settings_id.calendar_default_privacy',
+        readonly=False, required=True, groups="base.group_system"
+    )
```

Comment:
Is it necesary to add this related field? Couldn't we directly access the calendar_default_privacy value on the `res.users.settings` model ?